src/AISmart.Application.Contracts/GEvents/Social/SocialEvent.cs

```diff
@@ -13,4 +13,5 @@ public class SocialEvent:EventWithResponseBase<SocialResponseEvent>
     [Id(1)]  public string MessageId { get; set; }
     [Description("Unique identifier for the chat from which the message was received.")]
     [Id(2)]  public string ChatId { get; set; }
+    [Id(3)]  public string TweetId { get; set; }
```

Comment:
This should not be needed. Anyone receiving social event should not need to know about the tweet id. It seems like you want to know which response corresponds to which tweet id. You can use eventId that is returned when you PublishAsync and create a map between eventid and tweetid.